src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -9,9 +12,70 @@ interface IComponentProps {
   enablePiaIntakeForm: boolean;
 }
 function PPQLandingPage(props: IComponentProps) {
-  // TODO For ticket 402 use pia value to decide render pia intake or not
   const { enablePiaIntakeForm } = props;
-  return (
+  return enablePiaIntakeForm ? (
+    <div className="ppq-container background">
+      <div className="ppq-section results-wrapper">
+        <div className="row what-is-a-ppq">
+          <div className="col-md-6">
+            <h2 className="intake-heading">PIA Intake</h2>
+            <br />
+            <p>
+              The PIA Intake questions ask for some basic information about the
```

Comment:
No hardcoded text. Please use a separate message file. Refer any `messages.ts` for formatting.

Reply:
done